src/lib/y2partitioner/widgets/btrfs_subvolumes.rb

```diff
@@ -0,0 +1,64 @@
+require "yast"
+require "y2partitioner/widgets/btrfs_subvolumes_table"
+require "y2partitioner/widgets/btrfs_subvolumes_add_button"
+require "y2partitioner/widgets/btrfs_subvolumes_delete_button"
+require "y2partitioner/device_graphs"
+
+Yast.import "Mode"
+
+module Y2Partitioner
+  module Widgets
+    # Widget to manage btrfs subvolumes of a specific filesystem
+    #
+    # FIXME: How to handle events directly from a CWM dialog?
```

Comment:
well, ideally it should be handled by CWM, but maybe something is wrong here. It need some investigation.